<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite thread-count="1" name="Failed suite [Surefire suite]" verbose="0">
  <test thread-count="1" name="Surefire test(failed)" verbose="0">
    <classes>
      <class name="rahulshettyacademy.tests.ErrorValidationsTest">
        <methods>
          <include name="tearDown"/>
          <include name="ProductErrorValidation"/>
          <include name="launchApplication"/>
        </methods>
      </class> <!-- rahulshettyacademy.tests.ErrorValidationsTest -->
      <class name="cucumber.TestNGTestRunner">
        <methods>
          <include name="setUpClass"/>
          <include name="tearDownClass"/>
          <include name="runScenario" invocation-numbers="0"/>
        </methods>
      </class> <!-- cucumber.TestNGTestRunner -->
      <class name="rahulshettyacademy.tests.SubmitOrderTest">
        <methods>
          <include name="tearDown"/>
          <include name="OrderHistoryTest"/>
          <include name="submitOrder" invocation-numbers="0 1"/>
          <include name="launchApplication"/>
        </methods>
      </class> <!-- rahulshettyacademy.tests.SubmitOrderTest -->
    </classes>
  </test> <!-- Surefire test(failed) -->
</suite> <!-- Failed suite [Surefire suite] -->
